net.pistegamez.PekkaKana2.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Pekka Kana 2
+Comment=Jump 'n run game made in the spirit of old classic platformers
```

Comment:
I am not entirely sure whether the apostrophe character is valid in the Comment section of a XDG desktop file. However it does not seem to cause any harm, at least on my (GNOME-based) system.